dashboard/lib/script_constants.rb

```diff
@@ -131,7 +131,7 @@ module ScriptConstants
       MINECRAFT_HERO_NAME = 'hero'.freeze,
       MINECRAFT_NAME = 'mc'.freeze,
       MINECRAFT_DESIGNER_NAME = 'minecraft'.freeze,
-      MINECRAFT_AI_NAME = 'generation-ai'.freeze,
+      MINECRAFT_AI_NAME = 'generation_ai'.freeze,
```

Comment:
Can we change this back to a `-` at some point so that we match the format of all the other constants in this file?

We should be able to change the ID in the gsheet pretty easily, instead?